pkgs/servers/http/apache-modules/mod_ca/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, gnused, coreutils, pkgconfig, apacheHttpd, openssl, openldap }:
+
+stdenv.mkDerivation rec {
+ baseurl = "https://redwax.eu/dist/rs/";
+ name = "mod_ca";
+ suffix = ".tar.gz";
+
+ meta = with stdenv.lib; {
+   description = "RedWax CA service modules.";
+   version = "0.2.1";
+
+   homepage = "https://redwax.eu";
+   license = licenses.asl20;
+   platforms = platforms.unix;
+   maintainers = with maintainers; [ dirkx ];
+ };
+
+ src = fetchurl {
+   url = "${baseurl}${name}-${meta.version}${suffix}";
```

Comment:
As mentioned above it seems best to set this to `url = "https://redwax.eu/dist/rs/${pname}-${version}.tar.gz";` for this PR.